packages/web/src/pages/sign-up-page/components/MobileNavHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Box,
+  Flex,
+  IconAudiusLogoHorizontal,
+  IconCaretLeft,
+  PlainButton,
+  iconSizes
```

Comment:
should we put icon sizes is the theme?

Reply:
I went ahead and batched all the icon feedback together into a ticket
https://linear.app/audius/issue/C-3385/harmony-icon-components-tweaks